src/main/java/net/dancer/chatdancer/Message.java

```diff
@@ -0,0 +1,40 @@
+package net.dancer.chatdancer;
+
+import java.sql.Timestamp;
+
+public class Message {
```

Comment:
Under Developer the usage of Lombok is someway controversial, while my impression is that most like and use it. If you have no reasons to not use it, I would suggest you to give it a try ;-)

Reply:
Here I would annotate the class with an @Data

Reply:
After I added lombok to the pom file